flake.lock

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
I didn't notice this before, but could we move it to a separate PR or confirm that the dendrite/complement team are happy to maintain it?

Reply:
Oh lol, that shouldn't be there. I do want to PR some nix stuff soon to make my life easier, but not in this PR certainly.